app/controllers/suggestion.rb

```diff
@@ -0,0 +1,40 @@
+get '/suggestions' do
+  @suggestions = Suggestion.all
+  erb :"suggestions/index"
+end
+
+get '/suggestions/new' do
+  @suggestion = Suggestion.new
+  erb :"suggestions/index"
+end
+
+post '/suggestions' do
+  @suggestion = Suggestion.new(user_id: current_user.id, title: params[:suggestion][:title], details: params[:suggestion][:details])
```

Comment:
👍  This will work and also protects against spoofing (creating new records as someone else). But check this out: let's take advantage of your nested form parameters:
```ruby
@suggestion = Suggestion.new(params[:suggestion])
@suggestion.user = current_user
if @suggestion.save
  # etc.
```